accessibility-checker-engine/src/v2/checker/accessibility/rules/rpt-embed-rules.ts

```diff
@@ -85,6 +91,8 @@ let a11yRulesEmbed: Rule[] = [
             "dom:*[autostart=true], dom:*[autostart=1], dom:bgsound",
         run: (context: RuleContext, options?: {}): RuleResult | RuleResult[] => {
             const ruleContext = context["dom"].node as Element;
+            //skip the rule
+            if (RPTUtil.isNodeHidden(ruleContext)) return null;
```

Comment:
Can you explain why this is skipped? This is a concern with visible content. aria-hidden doesn't hide visible content.

Reply:
removed